client/src/pages/Anthony.jsx

```diff
@@ -2,11 +2,174 @@
+      <body>
+        <div className="header">
+          <div>
+            <img
+              className="headshot"
+              src={anthony}
+              alt="anthony headshot"
+              width="200"
+              height="200"
+            />
+          </div>
+          <div className="next-to-headshot">
+            <div className="name-in-header">
+              <h1 className="name-in-header">Anthony Cruz Macedo</h1>
+            </div>
+            <div>
+              <p>
+                <a
+                  className="link"
+                  href="https://www.facebook.com/profile.php?id=100072821919133"
+                >
+                  Facebook
+                </a>{' '}
+                <a className="link" href="https://www.instagram.com/saxytony/">
+                  {' '}
+                  Instagram
+                </a>{' '}
+                <a
+                  className="link"
+                  href="https://www.linkedin.com/in/acruzmacedo/"
+                >
+                  {' '}
+                  LinkedIn
+                </a>{' '}
+                <a
+                  className="link"
+                  href="https://open.spotify.com/user/3p6e4kte4mf77mujwzlgs6trw?si=b137c414428a4f00"
+                >
+                  Spotify
+                </a>
+              </p>
+            </div>
+          </div>
+        </div>
+        <div>
+          <h2>About Me</h2>
+          <h3 className="about-me-p">
+            My name is Anthony and I'm a sophomore at the University of Illinois
+            studying Computer Engineering. I am an Academy Member in Hack4Impact
+            and on the MemberSHPE Committee in the Society of Hispanic
+            Professional Engineers. I am excited to build and develop my skills
+            and knowledge during my time at UIUC!
+          </h3>
+        </div>
+        <div className="pictures">
+          <h2>Pictures</h2>
+          <div className="pictures-content">
+            <img className="memes" src={pic1} alt="pic1" />
+            <img className="memes" src={pic2} alt="pic2" />
+            <img className="memes" src={pic3} alt="pic3" />
+            <img className="memes" src={pic4} alt="pic4" />
+            <img className="memes" src={pic5} alt="pic5" />
+            <img className="memes" src={pic6} alt="pic6" />
+            <img className="memes" src={pic7} alt="pic7" />
+            <img className="memes" src={pic8} alt="pic8" />
+            <img className="memes" src={pic9} alt="pic9" />
+            <img className="memes" src={pic10} alt="pic10" />
+            <img className="memes" src={pic11} alt="pic11" />
+            <img className="memes" src={pic12} alt="pic12" />
+            <img className="memes" src={pic13} alt="pic13" />
+            <img className="memes" src={pic14} alt="pic14" />
+            <img className="memes" src={pic15} alt="pic15" />
+            <img className="memes" src={pic16} alt="pic16" />
```

Comment:
You can use a map function so you don't need to repeat code.

Create an array of all your images right before the return function (between line 40-41 right now), then here, do something like below where `images` is the name of the image array
```
{images.map((img, i) => (
     <img className="memes" src={img} alt={i} key={i} />
))}
```